### PR TITLE
fix bug where logging out and login again would not raise account changed event

### PR DESCRIPTION
**Pull request by wpeter-sfdc**
@kchitalia @smcnulty-sfdc

### CODE REVIEW

File: SalesforceSDK/Core/Auth/AccountManager.cs [hunk 1]
```diff
@@ -67,9 +67,6 @@ public static void DeleteAccount()
         {
             Account account = GetAccount();
             AuthStorageHelper.DeletePersistedCredentials(account.UserName, account.UserId);
-
-            // raise event about account logout
-            AuthenticatedAccountChanged?.Invoke(new AuthenticatedAccountChangedEventArgs(account, null));
```
Comment (kchitalia) [inline SalesforceSDK/Core/Auth/AccountManager.cs]: Dont you still need this here?

Comment (wpeter-sfdc) [inline SalesforceSDK/Core/Auth/AccountManager.cs]: no, when persisted credentials get deleted the event will get raised so we don't need to explicitly call it anymore here.

Comment (wpeter-sfdc) [inline SalesforceSDK/Core/Auth/AccountManager.cs]: the line right above this `AuthStorageHelper.DeletePersistedCredentials`, that's where the other change is that causes the property to get set which raises the event

**Comment by kchitalia** [comment]
:+1:

**Pull request merged**